### DISCUSSION TITLE
huggingface openLLM 跑分和 opencompass 的跑分不一致的原因是什么？

### DISCUSSION THREAD

**Discussion by amulil** [open-compass/opencompass#349] [View all discussions]
比如：

meta-llama/llama2-7b-chat-hf 的 mmlu 跑分

opencompass 结果是 46.2

huggingface openLLM 结果是 48.32

一样的模型，一样的数据集，造成这种差距的原因是什么

**Comment by tonysy** [comment]
Hi, the difference may be due to different prompts.
